addons/bestja_frontend_fpbz/views/pages.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <template id="layout_logo_show1" inherit_id="website.layout" customize_show="True" name="Show Logo1">
```

Comment:
Wydaje mi się, że większości bloków atrybuty "id" i "name" nie są adekwatne do zawartości. Popraw to jednak, bo one pełnią rolę również dokumentacji.